app/src/main/java/cm/aptoide/pt/app/DownloadStateParser.java

```diff
@@ -17,7 +17,7 @@ public DownloadStateParser() {
   public DownloadModel.DownloadState parseDownloadState(Install.InstallationStatus state,
       boolean isIndeterminate) {
     DownloadModel.DownloadState downloadState;
-    if (isIndeterminate) {
+    if (isIndeterminate && state != Install.InstallationStatus.INSTALLING) {
```

Comment:
The problem here is related to the Installing state. The installing state is visually a state where the indeterminate flag is true, but in this case, we don't want to map it to the DownloadState.INDETERMINATE, we want to map it to the DownloadState.INSTALLING. 
The problem here (and also the reason why Installing was visually broken) was because in the past we added this quick fix for the isIndeterminate, assuming that every time the isIndeterminate flag was true, we wanted to map it to the INDETERMINATE state.